Blog, Tauri Board Elections 2025 (#3372)
tauri-apps/tauri-docs: src/content/docs/blog/tauri-board-elections-2025.md

---
title: Tauri Board Elections 2025
date: 2025-06-27
authors: [beanow]
excerpt: As we continue to be amazed with all the applications you've created with Tauri v2, it's been 4 years that Tauri has become a Programme within The Commons Conservancy! This means we're preparing for a new round of Tauri Board Director elections for the next term.
---

> As we continue to be amazed with all the applications you've created with Tauri v2, it's been 4 years that Tauri has become a Programme within [The Commons Conservancy](https://commonsconservancy.org/)! This means we're preparing for a new round of Tauri Board Director elections for the next term.

Reviewer: ```suggestion
As we continue to be amazed by all the applications you've created with Tauri v2, it's been 4 years since  Tauri has become a Programme within [The Commons Conservancy](https://commonsconservancy.org/)! This means we're preparing for a new round of Tauri Board Director elections for the next term.
```

## Board Elections

> The Tauri Board of Directors is the central decision-making body for the Tauri Programme and is responsible for its overall health and stability. Additional details can be found on the [Tauri Governance page](/about/governance/).

Reviewer: ```suggestion
The Tauri Board of Directors is the central decision-making body for the Tauri Programme and is responsible for its overall health and stability. Additional details can be found at the [Tauri Governance page](/about/governance/).
```

In order to provide continuity from one year to another, the elections of seats are staggered over the course of 2 years. This means that one year a portion of the seats are elected, and the following year the remaining seats are elected. A Tauri Director seat is an elected position with a term of 2 years. The Board has a minimum of 3 Directors and a maximum of 7.

This year there are 5 seats open for election.

### Applying for Candidacy

There are 3 steps to express interest and apply for candidacy:

1. Learn about the role of a Tauri Board Director on the [Governance page](/about/governance/#tauri-board--board-directors).

> 2. Prepare a written introduction about yourself that covers who you are, your history and relevance for Tauri, and what you would like to bring to the Board. (Examples from previous years can be found on https://github.com/tauri-apps/governance-and-guidance).

Reviewer: ```suggestion
2. Prepare a written introduction about yourself that covers who you are, your history and relevance to Tauri, and what you would like to bring to the Board. (Examples from previous years can be found at https://github.com/tauri-apps/governance-and-guidance).
```
